website/pages/style.css

```diff
@@ -91,12 +92,3 @@
   background: var(--gray-1);
   color: white;
 }
-
-/** DEBT: short term patch until the branding changes roll out. Remove this with this PR — https://github.com/hashicorp/vault/pull/11118 **/
-.g-content {
-  --brand-text: var(--vagrant) !important;
-}
-.g-docs-sidenav {
-  --brand: var(--vagrant) !important;
-  --brand-text: var(--vagrant) !important;
-}
```

Comment:
Removed this as 1) PR in question, #11118, is merged, and 2) have double-checked that link styles look correct in docs-sidenav and in docs content.